ui/packages/consul-ui/app/services/hcp-link-status.js

```diff
@@ -3,17 +3,22 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import Service from '@ember/service';
+import Service, { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 
 const LOCAL_STORAGE_KEY = 'consul:hideHcpLinkBanner';
 
 export default class HcpLinkStatus extends Service {
+  @service abilities;
   @tracked
   userDismissedBanner = false;
 
-  get shouldDisplayBanner() {
-    return !this.userDismissedBanner;
+  get shouldDisplayHcpLinkPrompt() {
```

Comment:
Would love help with this variable name - `shouldDisplayBanner` doesn't cover it anymore - this property now correlates with:
- Displaying the purple banner
- Displaying the `new` tag in the nav item

Tried to correlate it to a linkPrompt - but now wondering if I should just make `bannerDismissed` and `hasPermissionToLink` available on the service and compute if the banner and `new` tag should be visible on their respective components instead of in this service - which probably shouldn't be too concerned with any UI logic - when you really think about it!

Would love opinions!